spaceship/lib/spaceship/connect_api/models/profile.rb

```diff
@@ -93,7 +93,9 @@ def self.create(client: nil, name: nil, profile_type: nil, bundle_id_id: nil, ce
           attributes: {
             name: name,
             profileType: profile_type,
-            templateName: template_name
+            # Unsupported
+            # https://developer.apple.com/documentation/appstoreconnectapi/profilecreaterequest/data-data.dictionary/attributes-data.dictionary
+            # templateName: template_name
```

Comment:
👋 Thanks a lot for this change! 

I don't think we need this comment. The git history and this pr should be enough to understand why we deleted it.

We also should clean up the callers and Sigh options, for example: 

```ruby 
Spaceship::ConnectAPI::Profile.create(..., template_name: Sigh.config[:template_name]) 
```